src/qt/utilitydialog.cpp

```diff
@@ -135,6 +136,19 @@ void HelpMessageDialog::on_okButton_accepted()
     close();
 }
 
+void HelpMessageDialog::setAboutWindowTitle(const NetworkStyle* networkStyle)
+{
+    QString aboutTitle = tr("About %1").arg(PACKAGE_NAME);
+    if ((networkStyle) && (Params().GetChainType() != ChainType::MAIN)) aboutTitle.append(" " + networkStyle->getTitleAddText());
```

Comment:
Putting a non-trivial expression into a separate line can facilitate debugging in the future:
```suggestion
    if (networkStyle && Params().GetChainType() != ChainType::MAIN) {
        aboutTitle.append(" " + networkStyle->getTitleAddText());
    }
```

Reply:
Ok, I'll update it, thanks!